pkg/retry/retry.go

```diff
@@ -103,7 +119,8 @@ func (s *Server) handleRetry() http.Handler {
 			}
 		}
 
-		logger.Infow("successful", "code", http.StatusAccepted, "body", acceptedMessage)
+		logger.Infow("successful", "code", http.StatusAccepted, "body", acceptedMessage, "totalEventCount", totalEventCount,
```

Comment:
Nit: when the log gets long, split it to KV pairs:

```suggestion
		logger.Infow("successful",
			"code", http.StatusAccepted,
			"body", acceptedMessage,
			"totalEventCount", totalEventCount,
```